src/webhooks/webhooks.ts

```diff
@@ -107,8 +108,9 @@ export class Webhooks {
   }
 
   async secureCompare(stringA: string, stringB: string): Promise<boolean> {
-    const bufferA = Buffer.from(stringA);
-    const bufferB = Buffer.from(stringB);
+    const encoder = new TextEncoder();
```

Comment:
Nit, but maybe this should be a class variable rather than being reinitialised.

Reply:
Good idea, done in [b0a8840](https://github.com/workos/workos-node/pull/965/commits/b0a8840ba59c1b57778f1e276db8eec889844e20)